README.md

```diff
@@ -0,0 +1,24 @@
+# README
+
+This README would normally document whatever steps are necessary to get the
+application up and running.
+
+Things you may want to cover:
+
+* Ruby version
+
+* System dependencies
+
+* Configuration
+
+* Database creation
+
+* Database initialization
+
+* How to run the test suite
+
+* Services (job queues, cache servers, search engines, etc.)
+
+* Deployment instructions
+
```

Comment:
- [ ] Please it will be nice to update your readme file to reflect the changes for this milestone, as this will make your readme file more informative and professional. Checkout this [template ](https://github.com/microverseinc/readme-template)from Microverse and adapt it to your project.

Reply:
Done